catalog.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="index.html">
+          <img src="" alt="Главная">
+        </a>
+        <ul>
+          <li>
+            <a href="">Информация</a>
+          </li>
+          <li>
+            <a href="">Фото и видео</a>
+          </li>
+          <li>
+            <a href="">Карта штата</a>
+          </li>
+          <li>
+            <a href="">Гостиницы</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="filter">
+        <fieldset>
+          <legend>Инфраструктура:</legend>
+          <ul>
+            <li>
+              <label for="check-1">
+                <input type="checkbox" checked="" name="check-1" id="check-1">
+                <span>Бассейн</span>
+              </label>
+            </li>
+            <li>
+              <label for="check-2">
+                <input type="checkbox" name="check-2" id="check-2">
+                <span>Парковка</span>
+              </label>
+            </li>
+             <li>
+              <label for="check-3">
+                <input type="checkbox" name="check-3" id="check-3">
+                <span>Wi-Fi</span>
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+         <fieldset>
+          <legend>Тип жилья:</legend>
+          <ul>
+            <li>
+              <label for="check-4">
+                <input type="checkbox" checked="" name="check-4" id="check-4">
+                <span>Гостиница</span>
+              </label>
+            </li>
+            <li>
+              <label for="check-5">
+                <input type="checkbox" checked="" name="check-5" id="check-5">
+                <span>Мотель</span>
+              </label>
+            </li>
+             <li>
+              <label for="check-6">
+                <input type="checkbox" checked="" name="check-6" id="check-6">
+                <span>Апартаменты</span>
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+        <fieldset>
+          <legend>Стоимость в сутки (Р):</legend>
+          <input type="range">
+          <button type="submit">Показать</button>
+        </fieldset>
+      </section>
+      <section class="content">
+        <div class="sort__top">
+          <h2>Найдено: 3</h2>
+          <ul>
+            <li>Сортировка:</li>
+            <li class="active">По цене</li>
+            <li>По типу</li>
+            <li>По рейтингу</li>
```

Comment:
типы сортировок это ссылки
```suggestion
            <li><a href="#!">По рейтингу</a></li>
```